st2common/st2common/models/api/action.py

```diff
@@ -497,6 +497,27 @@ class ActionAliasAPI(BaseAPI, APIUIDMixin):
                 "type": "array",
                 "items": {"type": "string"},
                 "description": "Possible parameter format."
+            },
+            "aliases": {
```

Comment:
@manasdk That's exactly what I considered, but I really don't like breaking backwards compatibility without a major release. What you're proposing is a great idea, but are we really fine with introducing something this drastic?